web_responsive/static/src/js/web_responsive.js

```diff
@@ -286,10 +289,24 @@ odoo.define('web_responsive', function(require) {
         new AppDrawer();
     });
 
+    // if we are in small screen change default view to kanban if exists
+    ViewManager.include({
+        get_default_view: function() {
+            var default_view = this._super()
+            if (config.device.size_class <= config.device.SIZES.XS &&
+                default_view.type != 'kanban' &&
+                this.views['kanban']){
```

Comment:
Totally stylistic, but it just took me a minute to trace these brackets. Please put the `{` on a new line, at same tab level as its closing bracket